Replace service.yaml with the correct contents (#1378)
knative/docs: docs/serving/samples/gitwebhook-go/README.md

## Before you begin

You must meet the following requirements to run this sample:

- Own a public domain. For example, you can create a domain with
  [Google Domains](https://domains.google/).
- A Kubernetes cluster running with the following:
  - Knative Serving must be installed. For details about
    setting up a Knative cluster, see the
    [installation guides](../../../install/README.md).
  - Your Knative cluster must be 
    [configured to use your custom domain](../../using-a-custom-domain.md).
  - You must ensure that your Knative cluster uses a static IP address:
    - For Google Kubernetes Engine, see 
      [assigning a static IP address](../../gke-assigning-static-ip-address.md).

>     - For other cloud provider, refer to your provider's documentation.

Reviewer: ```suggestion
    - For other cloud providers, refer to your provider's documentation.
```

Author: done

- An installed version of [Docker](https://www.docker.com).
- A [Docker Hub account](https://hub.docker.com/) to which you are able to
  upload your sample's container image.

## Build the sample code

1. Download a copy of the code:

    ```shell
    git clone -b "release-0.6" https://github.com/knative/docs knative-docs
    cd knative-docs/serving/samples/gitwebhook-go
    ```

> 1. Use Docker to build a container image for this service. Replace `username`

Reviewer: ```suggestion
1. Use Docker to build a container image for this service. Replace `{DOCKER_HUB_USERNAME}`
```

   with your Docker Hub username in the following commands.

    ```shell
    export DOCKER_HUB_USERNAME=username

    # Build the container, run from the project folder
    docker build -t ${DOCKER_HUB_USERNAME}/gitwebhook-go .

    # Push the container to the registry
    docker push ${DOCKER_HUB_USERNAME}/gitwebhook-go
    ```

> 1. Create a secret that holds two values from GitHub, a personal access token

Reviewer: ```suggestion
1. Create a secret that holds two values from GitHub:
```

>    used to make API requests to GitHub, and a webhook secret, used to validate

Reviewer: ```suggestion
    -  A personal access token that you will use to make API requests to GitHub
       - Ensure that you grant `read/write` permission in the repo for that personal access token.
```

  and a Docker Hub account configured (we'll use it for a container registry).
- An account on [GitHub](https://github.com) with read/write access to a
  repository.
- A public domain name available over the internet for your service to run this

>   example, if you use the repository in github.com to send a payload to your

Reviewer 2: s/github.com/[GitHub](https://github.com)

Author: done

>   webhook service. Check the guidance on how to get one for your service

Reviewer 2: Can you help add a link to the `guidance`?

Author: Each cloud provider should have its own guidance. I have not investigated all of them.

Reviewer 2: make sense, thanks.

Reviewer 3: Step 6 further down actually points to the doc about configuring a custom domain and assigning a static IP to it. I think we should bring that information to the sentence that you just added.

>       [create a personal access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/).

Reviewer: ```suggestion
    - A webhook secret that you will used to validate requests.
```

>       Ensure to grant the `repo` permission to give `read/write` access to the

Reviewer: ```suggestion
```

>       personal access token.

Reviewer: ```suggestion
```
